base_layer/core/src/base_node/chain_metadata_service/service.rs

```diff
@@ -85,19 +88,27 @@ impl ChainMetadataService {
             tokio::select! {
                 Ok(block_event) = block_event_stream.recv() => {
                     log_if_error!(
-                        level: debug,
+                        level: warn,
```

Comment:
Maybe `info` ?

From an outsiders perspective there's a high likelihood at any time they could receive data with unserailizable information but showing them big red warnings in the console makes them feel like they need to do something, when in fact this is a perfectly fine event to happen, and there's not actually any user required interaction that needs to occur.

Reply:
I'll meet you halfway with `info`. I think it is an indication that you are using the wrong build or network